src/components/AddressSearch/index.tsx

```diff
@@ -291,36 +174,36 @@ function AddressSearch({
         if (!values.street && details.adr_address) {
             const streetAddressRegex = /<span class="street-address">([^<]*)<\/span>/;
             const adr_address = details.adr_address.match(streetAddressRegex);
-            const streetAddressFallback = lodashGet(adr_address, [1], null);
+            const streetAddressFallback = adr_address ? adr_address[1] : null;
             if (streetAddressFallback) {
                 values.street = streetAddressFallback;
             }
         }
 
         // Not all pages define the Address Line 2 field, so in that case we append any additional address details
         // (e.g. Apt #) to Address Line 1
-        if (subpremise && typeof renamedInputKeys.street2 === 'undefined') {
+        if (subpremise && typeof renamedInputKeys?.street2 === 'undefined') {
             values.street += `, ${subpremise}`;
         }
 
-        const isValidCountryCode = lodashGet(CONST.ALL_COUNTRIES, country);
+        const isValidCountryCode = Object.keys(CONST.ALL_COUNTRIES).find((foundCountry) => foundCountry === country);
         if (isValidCountryCode) {
             values.country = country;
         }
 
         if (inputID) {
-            _.each(values, (inputValue, key) => {
-                const inputKey = lodashGet(renamedInputKeys, key, key);
+            Object.keys(values).forEach((inputValue, key) => {
+                const inputKey = renamedInputKeys?.key ?? key;
```

Comment:
Please have a look at other lodashGets:
```suggestion
                const inputKey = renamedInputKeys[key] ?? key;
```